app/models/form/question.rb

```diff
@@ -191,6 +191,25 @@ def suffix_label(case_log)
     label
   end
 
+  def answer_option_synonyms(answer_id)
```

Comment:
Not sure how performant this'll be. We're doing (3 * number of active schemes in the organisation) + 1 database calls here just to render the type ahead question. Might be better to have these methods take a model as it's argument and expect the model to provide the hint, and synonym values.

That way you can have a single database call to fetch all the required records, and it's easier to extend these methods to other things than schemes without building up big if/case statements?